src/common/types/get-query-response.type.ts

```diff
@@ -3,6 +3,7 @@ type GetQueryResponse<T> = {
 	next: null | number;
 	previous: null | number;
 	results: T[];
+	reviews_count: number;
```

Comment:
it's common type for every get request. only companies and courses includes review_count. Maybe better make it optional